pkg/certifier/osv/osv.go

```diff
@@ -85,6 +89,7 @@ func (o *osvCertifier) CertifyComponent(ctx context.Context, rootComponent inter
 	if err != nil {
 		return fmt.Errorf("osv.dev batched request failed: %w", err)
 	}
```

Comment:
can we add a test to make sure that a ratelimit fails gracefully?

Reply:
I have included tests for osv, deps dev, and clearly defined.